.github/workflows/ios-test.yaml

```diff
@@ -0,0 +1,47 @@
+name: Test-iOS
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - '*'
+jobs:
+  ios-test:
+    runs-on: macos-latest
+
+    env:
+      FLUTTER_CHANNEL: stable
+      FLUTTER_VERSION: 3.22.2
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: fetch submodules
+        run: git submodule update --init --recursive
+
+      - name: Copy iOS
+        working-directory: ios/Classes
+        run: cp -r confidence-sdk/Sources/Confidence .
+
+      - name: Remove the submodule
+        working-directory: ios/Classes
+        run: rm -rf confidence-sdk
+
+      - name: Remove git submodule
+        working-directory: ios/Classes
+        run: git rm confidence-sdk
+
+      - uses: futureware-tech/simulator-action@v3
```

Comment:
This action only have 44 stars, should we look for something more well used?

Reply:
this is the most famous one I found. we can change it if it becomes flaky in the future or what is your concern?